src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension/src/extension.ts

```diff
@@ -20,7 +20,7 @@ export async function activate(context: vscode.ExtensionContext) {
     const config = process.env.config ? process.env.config : 'Debug';
 
     const languageServerDir = path.join(
-        __dirname, '..', '..', '..', '..', '..', 'artifacts', 'bin', 'rzls', config, 'net7.0');
+        __dirname, '..', '..', '..', '..', '..', 'artifacts', 'bin', 'rzls', config, 'net8.0');
```

Comment:
@allisonchou is this right? I don't know how we decide which TFM to use to publish rzls, but it would be surprising to me if this worked without _something_ else changing. We also probably want to be deliberate about which TFM we do publish.

Reply:
> if this worked without _something_ else changing

You were right, it didn't work. I changed something else:

https://github.com/dotnet/razor/blob/21b656261cf842e444e174b18395656664f957a6/src/Razor/src/rzls/rzls.csproj#L5-L6

Reply:
Not sure that will work either.. AFAIK, what `TargetFrameworks` does is run multiple builds, with each one setting the `TargetFramework` to the right value, in turn, from the set that is specified. Specifying both doesn't make sense. To control publishing I would imagine we would need to pass in the target framework as an option to the publish command, but I don't know where that lives. I'm also not sure we want to update it to NET 8 at this point.

Reply:
That was my understanding as well, but it actually fixed the errors on Linux. But broke Windows build. Maybe we can have just single TFM for rzls?

Reply:
This controls the VS code extension correct? If so we likely want to keep that on the RTM .NET TFM so that would be `net7.0` here. I _believe_ moving this to `net8.0` essetnially forces the VS code extension to `net8.0` which doesn't seem to be what we want here.